### DISCUSSION TITLE
Dataview query emoji table issue

### DISCUSSION THREAD

**Discussion by vizance** [BryanJenksCommunity/FAQ#62] [Q&A] [answered]
I'm wondering how to use emoji tag in table query. Something like #🗺️  won't work if I write 
```dataview
table field
from #🗺️ 
```
Obsidian will show below error message.
```
> 2 | from #<img draggable="false" class="emoji" alt="🗺️" src="https://twemoji.maxcdn.com/v/13.0.1/72x72/1f5fa.png"> 
    |      ^
  3 | 

Expected one of the following: 

'!', '(', '-', 'outgoing(', file link, string, tag ('#hello/stuff')
```

Thanks

**Comment by tallguyjenks** [comment, marked as answer]
yes, this is an issue i reported on the dataview repo. in the `FROM` clause for tags, Emoji's are causing errors, my temporary work around thus far has been to use this:

````
```dataview
list
from ""
where tags="🗺️"
```
````
